feat(vpn-port-forwarding): add qbittorrent port update example (#121)
qdm12/gluetun-wiki: setup/advanced/vpn-port-forwarding.md

- one can bind mount a shell script in Gluetun and execute it with for example `VPN_PORT_FORWARDING_UP_COMMAND=/bin/sh -c /gluetun/myscript.sh` - 💁  feel free to propose a pull request to add commonly used shell scripts for port forwarding!
- the output of the command is written to the port forwarding logger within Gluetun

> ### qBittorrent Example

Reviewer: ```suggestion
### qBittorrent example
```

> `VPN_PORT_FORWARDING_UP_COMMAND=/bin/sh -c '/usr/bin/wget -O- --retry-connrefused --post-data "json={\"listen_port\":{{PORTS}}}" http://127.0.0.1:8080/api/v2/app/setPreferences 2>&1'`

Reviewer: ```suggestion
`VPN_PORT_FORWARDING_UP_COMMAND=/bin/sh -c 'wget -O- --retry-connrefused --post-data "json={\"listen_port\":{{PORTS}}}" http://127.0.0.1:8080/api/v2/app/setPreferences 2>&1'`
```

> - In order to get the call working make sure port qBittorrent is listening on is open. IE it should be added as port being exposed by gluetun. In the example above we are assuming port `8080` is the port being used by qBittorrent.
> - Add `127.0.0.1/32` to bypass authentication settings for qBittorrent.

Reviewer: ```suggestion
For this to work, the qBittorrent web UI server must be enabled and listening on port `8080` and the Web UI "Bypass authentication for clients on localhost" must be ticked (json key `bypass_local_auth`) so Gluetun can reach qBittorrent without authentication.
```
